--- ipsec-tools-0.8.1/configure.ac.orig	2013-07-11 12:22:31.437559080 -0500
+++ ipsec-tools-0.8.1/configure.ac	2013-01-08 06:43:29.000000000 -0600
@@ -388,6 +388,7 @@
     [libpam_dir=$withval], 
     [libpam_dir=no])
 AC_MSG_RESULT($libpam_dir)
+LIBS_libpam=""
 if test "$libpam_dir" != "no"; then
 	if test "$libpam_dir" = "yes" ; then
 		  libpam_dir="";
@@ -400,12 +399,13 @@
 		    -d "$libpam_dir/include" ; then
 			RACOON_PATH_LIBS([pam_start], [pam], ["$libpam_dir/lib"])
 			CPPFLAGS_ADD="$CPPFLAGS_ADD -I$libpam_dir/include"
+			LIBS_libpam="-L$libpam_dir/lib -R$libpam_dir/lib"
 		else
 			AC_MSG_ERROR([PAM libs or includes not found. Aborting.])
 	  	fi
 	fi
 	AC_DEFINE([HAVE_LIBPAM], [], [Hybrid authentication uses PAM])
+	LIBS="$LIBS $LIBS_libpam -lpam"
-	LIBS="$LIBS -L$libpam_dir/lib -R$libpam_dir/lib -lpam"
 	AC_CHECK_FUNCS(pam_start)
 fi
 
